contracts/actions/liquity/LiquitySPDeposit.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "./helpers/LiquityHelper.sol";
+import "../../utils/TokenUtils.sol";
+import "../ActionBase.sol";
+
+contract LiquitySPDeposit is ActionBase, LiquityHelper {
+    using TokenUtils for address;
+
+    struct Params {
+        uint256 lusdAmount;
+        address from;
+        address wethTo;
+        address lqtyTo;
+    }
+
+    /// @inheritdoc ActionBase
+    function executeAction(
+        bytes[] memory _callData,
+        bytes[] memory _subData,
+        uint8[] memory _paramMapping,
+        bytes32[] memory _returnValues
+    ) public payable virtual override returns (bytes32) {
+        Params memory params = parseInputs(_callData);
+        params.lusdAmount = _parseParamUint(params.lusdAmount, _paramMapping[0], _subData, _returnValues);
+        params.from = _parseParamAddr(params.from, _paramMapping[1], _subData, _returnValues);
+        params.wethTo = _parseParamAddr(params.wethTo, _paramMapping[2], _subData, _returnValues);
```

Comment:
Can be just one addr and send both to same addr

Reply:
non issue

Reply:
non issue